ssh-keygen.c

```diff
@@ -241,10 +241,8 @@ type_bits_valid(int type, const char *name, u_int32_t *bitsp)
 	if (IS_ECDSA_HYBRID(type)) {
 		switch (type) {
 ///// OQS_TEMPLATE_FRAGMENT_SET_BITS_START
```

Comment:
Do we need this switch statement now, since it looks like all cases were for rainbow?

Reply:
This code is caused by an error in the oqs-template jinja script. Given this is for code we're effectively not maintaining any more, let me see whether it's simple to change...

Reply:
It was. See new commit.